ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -125,10 +125,15 @@ export const toAppType = (type: UIAppType): AppType =>
     [UIAppType.RSTUDIO, () => AppType.RSTUDIO]
   );
 
+export const toUIAppType: Record<AppType, UIAppType> = {
```

Comment:
this is a nicer approach - would you mind converting `toUIAppType` to match?

Reply:
Done 👍